code/modules/reagents/chemistry/recipes/pyrotechnics.dm

```diff
@@ -56,7 +56,7 @@
 /datum/chemical_reaction/nitroglycerin
 	name = "Nitroglycerin"
 	id = "nitroglycerin"
-	result = "nitroglycerin"
+	result = null
```

Comment:
Tested PR locally and it works, but please remove this line since it defaults to a null result
```dm
/datum/chemical_reaction
	var/name = null
	var/id = null
	var/result = null
```

Reply:
done